### PR TITLE
Add discord invite to contributing doc

### PR DESCRIPTION
**Pull request by ggwpez**
Changes:
- Convert `CONTRIBUTING`  from  adoc to markdown.
- Add Discord badge to `CONTRIBUTING.md`.

Can we add a direct channel invite link here? Not sure how Discord is normally used.

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# Contributing
+
+The `Substrate` project is an ***OPENISH Open Source Project***
+
+Contributors are invited to our `#frame-contributors` channel on the Polkadot Discord for support and coordination:  
+[![Discord](https://img.shields.io/discord/722223075629727774?style=for-the-badge&logo=discord&label=Discord)](https://dot.li/discord)
```
Comment (ggwpez) [inline docs/CONTRIBUTING.md:6]: This was added

**Review by liamaharon — APPROVED** [review]

**Review by franciscoaguirre — APPROVED** [review]
Awesome

**Review by kianenigma — APPROVED** [review]
Taking it a step further, we can also add it here https://paritytech.github.io/ cc @wentelteefje

**Review by gpestana — APPROVED** [review]

**Comment by gilescope** [comment]
Fine with it but once we merge we will need to fixup the broken links: https://github.com/search?q=org%3Aparitytech%20CONTRIBUTING.adoc&type=code

**Comment by ggwpez** [comment]
bot merge

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (bkchr) [reply docs/CONTRIBUTING.md:6]: Not sure why we bring back a channel that we deprecated a year ago and then we also do this on discord...

Comment (ggwpez) [reply docs/CONTRIBUTING.md:6]: Its for making the external FRAME contributors feel more included for getting hints about their issues and coordination.  
All technical Qs will still be redirected to SE etc.

**Pull request merged**